src/analysis/mean_inequalities.lean

```diff
@@ -230,7 +230,8 @@ begin
   have h := rpow_arith_mean_le_arith_mean_rpow (univ : finset (fin 2))
     (fin.cons w₁ $ fin.cons w₂ fin_zero_elim) (fin.cons z₁ $ fin.cons z₂ $ fin_zero_elim) _ hp,
   { simpa [fin.sum_univ_succ, fin.sum_univ_zero, fin.cons_succ, fin.cons_zero] using h, },
-  { simp [hw', fin.sum_univ_succ, fin.sum_univ_zero, fin.cons_succ, fin.cons_zero], },
+  { simp [hw', fin.sum_univ_succ, fin.sum_univ_zero, fin.cons_succ, fin.cons_zero,
+          -unique_has_one] }
```

Comment:
This makes me worry a little that `unique_has_one` might be a bad simp lemma. What does the goal get stuck at if you don't add this line?